exporter/honeycombexporter/honeycomb.go

```diff
@@ -16,91 +16,287 @@ package honeycombexporter
 
 import (
 	"context"
+	"time"
 
-	"github.com/honeycombio/opentelemetry-exporter-go/honeycomb"
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	libhoney "github.com/honeycombio/libhoney-go"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/component/componenterror"
 	"go.opentelemetry.io/collector/consumer/consumerdata"
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
-	"go.opentelemetry.io/otel/api/core"
+	"go.uber.org/zap"
 )
 
-const oTelCollectorUserAgentStr = "Honeycomb-OpenTelemetry-Collector"
+// User agent string to use when sending events to Honeycomb.
+const (
+	oTelCollectorUserAgentStr = "Honeycomb-OpenTelemetry-Collector"
+	oTelCollectorVersionStr   = "0.3.0"
```

Comment:
Better to get the version number from `version.Version`.

Reply:
I tried that, unfortunately ran into some build problems importing that package:

```
go: finding module for package github.com/open-telemetry/opentelemetry-collector-contrib/internal/version
go: found github.com/open-telemetry/opentelemetry-collector-contrib/internal/version in github.com/open-telemetry/opentelemetry-collector-contrib v0.3.0
go: github.com/open-telemetry/opentelemetry-collector-contrib@v0.3.0 requires
	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter@v0.0.0: reading github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter/exporter/awsxrayexporter/go.mod at revision exporter/awsxrayexporter/v0.0.0: unknown revision exporter/awsxrayexporter/v0.0.0
make: *** [test] Error 1
```

Because it's an internal package, I can't use the one in `opentelemetry-collector` and have to depend on the contrib repo. Any ideas?

Reply:
I see. There is no trivial workaround. Perhaps drop the version number for now since it will very likely go stale very soon when bump the version number. Up to you.

Reply:
> I see. There is no trivial workaround. Perhaps drop the version number for now since it will very likely go stale very soon when bump the version number. Up to you.

Good call, removed.